examples/python/test.sh

```diff
@@ -0,0 +1,15 @@
+# Copyright 2022 Science project contributors.
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+# shellcheck source=../common.sh
+source "${COMMON}"
+trap gc EXIT
+
+"${SCIE_JUMP}" "lift.${OS_ARCH}.json"
+gc "${PWD}/pants" "${PWD}/.pants.d" "${PWD}/.pids"
+
+time RUST_LOG=trace ./pants --no-pantsd -V
+time ./pants --no-pantsd -V
```

Comment:
Any reason this is run twice?

Reply:
Just to show the speedup after a 1st run extraction.

Reply:
👍🏽 Excellent